robot_skills/src/robot_skills/head.py

```diff
@@ -22,59 +20,6 @@
 from .util.kdl_conversions import kdlVectorStampedToPointStamped, VectorStamped
 
 
-WavingResult = namedtuple('WavingResult', ['side', 'roi'])
-
-
-class Skeleton(object):
```

Comment:
I see this bit removed but not added again anywhere? We don't need it anymore? @Rayman any ideas?

Reply:
I couldn't find any usage of it. Any a new skeleton class is used in people_detector

Reply:
This is true, it's all in the people detector package

Reply:
OK then